### PR TITLE
test(e2e/tst_communityPermissions): New tests case `tst_communityPermissions`

### LINKED ISSUE #9007: [Community Permissions - Squish] Add `Welcome permission screen` user story
### Description
Add into `suite_communities/tst_communityPermissions` the following user story:

```gherkin
Background:

	Given A first time user lands on the status desktop and generates new key
	And the user signs up with username "tester123" and password "TesTEr16843/!@00"
	And the user lands on the signed in app
	And the user opens the community portal section
        And the user lands on the community portal section
        And the user creates a community named "Test-Community", with description "My community description", intro "Community Intro" and outro "Community Outro"
         And the user lands on the community named "Test-Community"
	And the user opens app settings screen
	And Application Settings "Advanced" is open
	And "Community Permissions Settings" is toggled on under Experimental features
	And the user opens the communities settings
	And the user opens the community named "Test-Community"
	And the user lands on the community named "Test-Community"
	And "Manage Community" is clicked in the community sidebar
	And "Permissions" should be an available option in Community Settings

Scenario: Welcome Permissions Screen content validation
    When "Permissions" section is selected
    Then the heading is "Permissions"
    And the welcome permission image is present
    And the welcome permission title is present
    And the welcome permission subtitle is present
    And the welcome permission settings checklist is present
    And Add new permission button is present
```

**NOTE**: Here the user story definition in [Notion](https://www.notion.so/Community-permissions-ab5960ce480243b9a4db3b4a09d11530#81d8f6b674e549279feb7f4fc8372899)

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #9007

### What does the PR do

- Created new test case for community permissions.
- Added new scenario "Welcome Permissions Screen content validation"

### Affected areas
e2e/tst_communityPermissions

### CODE REVIEW

**Review by glitchminer — APPROVED** [review]

**Review by glitchminer — APPROVED** [review]

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ class CommunityScreenComponents(Enum):
     EXISTING_CONTACTS_LISTVIEW = "community_InviteFirends_Popup_ExistinContacts_ListView"
     INVITE_POPUP_NEXT_BUTTON = "community_InviteFriendsToCommunityPopup_NextButton"
     INVITE_POPUP_MESSAGE_INPUT = "community_ProfilePopupInviteMessagePanel_MessageInput"
-    INVITE_POPUP_SEND_BUTTON = "community_InviteFriend_SendButton"
+    INVITE_POPUP_SEND_BUTTON = "community_InviteFriend_SendButton"
```
Comment (jrainville) [inline test/ui-test/src/screens/StatusCommunityScreen.py:57]:
```suggestion
    INVITE_POPUP_SEND_BUTTON = "community_InviteFriend_SendButton"
```

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 2]
```diff
@@ -465,4 +476,37 @@ def click_sidebar_option(self, community_sidebar_option:str):
...
+        
+    def verify_welcome_settings_checklist(self):
+        checklist = []
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT1.value).text)
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT2.value).text)
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT3.value).text)
+
+        verify_equals(checklist[0], "Give individual members access to private channels")
+        verify_equals(checklist[1], "Monetise your community with subscriptions and fees")
+        verify_equals(checklist[2], "Require holding a token or NFT to obtain exclusive membership rights")
```
Comment (jrainville) [inline test/ui-test/src/screens/StatusCommunityScreen.py:508]: In my opinion, testing text can be flimsy. If the text is changed, we have to update the tests as well. Maybe just checking if the text obj is there and visible is enough?

Comment (noeliaSD) [reply test/ui-test/src/screens/StatusCommunityScreen.py:508]: I think that in some point we will also need to check the texts and also text in different languages so.. But maybe an improvement here would be to add it as a param in the BDD feature file!

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 3]
```diff
@@ -465,4 +476,37 @@ def click_sidebar_option(self, community_sidebar_option:str):
...
+    
+    def verify_add_permission_button_present(self):
+        button_title = get_obj(CommunityPermissionsComponents.ADD_NEW_PERMISSION_BUTTON.value).text
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusCommunityScreen.py:511]: WDYT if we verify if the button is `enabled` instead??

File: test/ui-test/src/screens/StatusCommunityScreen.py (hunk 2 above)
Comment (jrainville) [reply test/ui-test/src/screens/StatusCommunityScreen.py:508]: Sounds good

File: test/ui-test/testSuites/suite_communities/tst_communityPermissions/test.feature [hunk 4]
```diff
@@ -8,8 +8,19 @@ Background:
...
-   Then "Permissions" should be an available option in Community Settings
\ No newline at end of file
+ Scenario: Welcome Permissions Screen content validation
+    When "Permissions" section is selected
+    Then the heading is "Permissions"
+    And the welcome permission image is present
+    And the welcome permission title is present
+    And the welcome permission subtitle is present
+    And the welcome permission settings "<user onboarding checklist>" is present
+   	| Give individual members access to private channels |
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_communities/tst_communityPermissions/test.feature:22]: Thanks for changing it to a table!! :)

**Review by anastasiyaig — APPROVED** [review]

**Review by StateOf-Vlado — APPROVED** [review]

**Review by glitchminer — APPROVED** [review]

File: test/ui-test/testSuites/suite_communities/shared/scripts/community_names.py [hunk 5]
```diff
@@ -81,3 +81,18 @@
...
+communityPermissions_welcome_title = {"container": statusDesktop_mainWindow, "objectName": "welcomeSettingsTitle", "type": "StatusBaseText", "visible": True}
+communityPermissions_welcome_image = {"container": statusDesktop_mainWindow, "objectName": "welcomeSettingsImage", "type": "Image", "visible": True}
+communityPermissions_welcome_settings_title = {"container": statusDesktop_mainWindow, "objectName": "welcomeSettingsTitle", "type": "StatusBaseText", "visible": True}
+communityPermissions_welcome_settings_subtitle = {"container": statusDesktop_mainWindow, "objectName": "welcomeSettingsSubtitle", "type": "StatusBaseText", "visible": True}
+communityPermissions_welcome_settings_checkList_element1 = {"container": statusDesktop_mainWindow, "objectName": "checkListText_0", "type": "StatusBaseText", "visible": True}
+communityPermissions_welcome_settings_checkList_element2 = {"container": statusDesktop_mainWindow, "objectName": "checkListText_1", "type": "StatusBaseText", "visible": True}
+communityPermissions_welcome_settings_checkList_element3 = {"container": statusDesktop_mainWindow, "objectName": "checkListText_2", "type": "StatusBaseText", "visible": True}
```
[the remark this answers is not in the document]
Comment (anastasiyaig) [reply test/ui-test/testSuites/suite_communities/shared/scripts/community_names.py:92]: this screen wont change i hope (no more options expected to be added). I will leave it as is for now, but we can improve any time

**Pull request merged**